nim/tests/test_routes.py

```diff
@@ -0,0 +1,149 @@
+import json
+
+import pytest
+
+from nim.game import (
+    DEFAULT_MIN,
+    DEFAULT_MAX,
+    DEFAULT_PILES
+)
+
+from nim.app import app
+
+
+@pytest.fixture
+def client_app():
+    return app.test_client()
+
+
+def assert_new_game_success(passed_client, req):
+    """
+    Asserts post to /new is 200
+    & state is not empty
+    Checks piles is in line with req &
+    none in response lower than mix or
+    higher than max
+    :param passed_client: test_client
+    :param req: Desired game conditions
+    """
+    response = passed_client.post(
+        '/new',
+        content_type='application/json',
+        data=req
+    )
+    assert response.status_code == 200
+    response = json.loads(response.data)
+    request = json.loads(req)
+    assert 'state' in response
+    state = response['state']
+    assert all(isinstance(x, int) for x in state)
+    assert request.get('piles', DEFAULT_PILES) == len(state)
+    assert request.get('min', DEFAULT_MIN) <= min(state) <= max(state)
+    request.get('max', DEFAULT_MAX)
```

Comment:
Look at how lonely this line is, not doing anything :stuck_out_tongue:

Reply:
I sawwwww :(